src/Service.Tests/Configuration/ConfigurationTests.cs

```diff
@@ -2030,6 +2030,155 @@ public async Task ValidateErrorMessageForMutationWithoutReadPermission()
             }
         }
 
+        /// <summary>
+        /// Multiple mutation operations are disabled through the configuration properties.
+        ///
+        /// 1. Executing the following multiple create mutation operations,
+        ///     a. Point multiple create operation
+        ///     b. Many type multiple create operation
+        ///    should result in failure as both these operations are disabled.
+        ///
+        /// 2. Executing a create mutation operation should succeed.
+        /// 
+        /// </summary>
+        [TestMethod]
+        [TestCategory(TestCategory.MSSQL)]
+        public async Task ValidateMultipleCreateAndCreateMutationWhenMultipleCreateOperationIsDisabled()
+        {
+            // Multiple create operations are disabled.
+            GraphQLRuntimeOptions graphqlOptions = new(Enabled: true, MultipleMutationOptions: new(new(enabled: false)));
+
+            RestRuntimeOptions restRuntimeOptions = new(Enabled: false);
+
+            DataSource dataSource = new(DatabaseType.MSSQL,
```

Comment:
super-nit: Will be more clear to either write all of the params together or in separate lines.